.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,39 +1,40 @@
-Fixes #
-
-# Description
-
-Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
-
-## Type of change
-
-Please mark relevant options with an `x` in the brackets.
-
-- [ ] Bug fix (non-breaking change which fixes an issue)
-- [ ] New feature (non-breaking change which adds functionality)
-- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-- [ ] This change requires a documentation update
-- [ ] Algorithm update - updates algorithm documentation/questions/answers etc.
-- [ ] Other (please describe):
-
-# How Has This Been Tested?
-
-Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
-
-- [ ] Integration tests
-- [ ] Unit tests
-- [ ] Manual tests 
-- [ ] No tests required
-
-# Reviewer checklist
-
-Mark everything that needs to be checked before merging the PR.
-
-- [ ] Check if the code is well documented
-- [ ] Check if the behavior is what is expected
-- [ ] Check if the code is well tested
-- [ ] Check if the code is readable and well formatted
-- [ ] Additional checks (document below if any)
-
-# Screenshots (if appropriate):
-
-# Questions (if appropriate):
\ No newline at end of file
+Fixes #
+
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+## Type of change
+
+Please mark relevant options with an `x` in the brackets.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires documentation changes (link at least one documentation issue):
+- [ ] Algorithm update - updates algorithm documentation/questions/answers etc.
+- [ ] Other (please describe):
+
+# How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
+
+- [ ] Integration tests
+- [ ] Unit tests
+- [ ] Manual tests 
+- [ ] No tests required
+
+# Reviewer checklist
+
+Mark everything that needs to be checked before merging the PR.
+
+- [ ] Check if the code is well documented
+- [ ] Check if the behavior is what is expected
+- [ ] Check if the code is well tested
+- [ ] Check if the code is readable and well formatted
+- [ ] Additional checks (document below if any)
+- [ ] Check if documentation update issue(s) are created if the option `This change requires a documentation update` above is selected

```

Comment:
same here, this is just a couple of lines that were changed in the PK-Sim repository version.